### PR TITLE
GEOPY-689

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-689 - Re-label 2D inversion menu**

### CODE REVIEW

File: geoapps/inversion/electricals/direct_current/two_dimensions/constants.py [hunk 1]
```diff
@@ -282,6 +322,46 @@
...
+    },
+    "receivers_offset_x": {
+        "group": "Data pre-processing",
+        "label": "Receiver X offset (m)",
+        "optional": True,
+        "enabled": False,
+        "value": 0.0,
+        "visible": False,
+    },
+    "receivers_offset_y": {
```
Comment (domfournier) [inline geoapps/inversion/electricals/direct_current/two_dimensions/constants.py:338]: Thought we got rid of those?

File: geoapps/inversion/electricals/induced_polarization/three_dimensions/constants.py [hunk 2]
```diff
@@ -14,15 +14,17 @@
 from geoapps.inversion import default_ui_json as base_default_ui_json
 
 inversion_defaults = {
-    "title": "SimPEG Induced Polarization inversion",
+    "title": "SimPEG Induced Polarization 3d inversion",
```
Comment (domfournier) [inline geoapps/inversion/electricals/induced_polarization/three_dimensions/constants.py:17]:
```suggestion
    "title": "SimPEG Induced Polarization 3D inversion",
```

File: geoapps/inversion/electricals/induced_polarization/three_dimensions/constants.py [hunk 3]
```diff
@@ -145,6 +147,17 @@
 }
 
 forward_ui_json = {
+    "starting_model": {
+        "association": ["Cell", "Vertex"],
```
Comment (domfournier) [inline geoapps/inversion/electricals/induced_polarization/three_dimensions/constants.py:151]:
```suggestion
        "association": "Cell",
```

File: geoapps/inversion/electricals/direct_current/three_dimensions/constants.py [hunk 4]
```diff
@@ -148,6 +146,17 @@
 }
 
 forward_ui_json = {
+    "starting_model": {
+        "association": ["Cell", "Vertex"],
```
Comment (domfournier) [inline geoapps/inversion/electricals/direct_current/three_dimensions/constants.py:150]:
```suggestion
        "association": "Cell",
```

File: geoapps/inversion/electricals/induced_polarization/three_dimensions/constants.py [hunk 5]
```diff
@@ -252,6 +274,42 @@
...
     },
+    "receivers_offset_x": {
+        "group": "Data pre-processing",
+        "label": "Receiver X offset (m)",
+        "optional": True,
+        "enabled": False,
+        "value": 0.0,
+        "visible": False,
+    },
+    "receivers_offset_y": {
```
Comment (domfournier) [inline geoapps/inversion/electricals/induced_polarization/three_dimensions/constants.py:285]: Same here remove offset_x and offset_y

File: geoapps/inversion/electricals/induced_polarization/two_dimensions/constants.py [hunk 6]
```diff
@@ -95,23 +96,23 @@
...
     "workspace_geoh5": None,
     "run_command": "geoapps.inversion.driver",
-    "run_command_boolean": False,
     "conda_environment": "geoapps",
     "distributed_workers": None,
     "chargeability_channel_bool": True,
 }
 forward_defaults = {
-    "title": "SimPEG Induced Polarization Forward",
+    "title": "Induced Polarization (IP) 2D forward",
```
Comment (domfournier) [inline geoapps/inversion/electricals/induced_polarization/two_dimensions/constants.py:104]:
```suggestion
    "title": "Induced Polarization (IP) 2D Forward",
```

File: geoapps/inversion/electricals/induced_polarization/two_dimensions/constants.py [hunk 7]
```diff
@@ -344,6 +387,42 @@
...
     },
+    "receivers_offset_x": {
+        "group": "Data pre-processing",
+        "label": "Receiver X offset (m)",
+        "optional": True,
+        "enabled": False,
+        "value": 0.0,
+        "visible": False,
+    },
+    "receivers_offset_y": {
```
Comment (domfournier) [inline geoapps/inversion/electricals/induced_polarization/two_dimensions/constants.py:398]: Same here - remove horizontal offsets

File: geoapps/inversion/potential_fields/gravity/constants.py [hunk 8]
```diff
@@ -123,7 +122,7 @@
     "gy_channel_bool": False,
 }
 forward_defaults = {
-    "title": "Gravity Forward",
+    "title": "Gravity forward",
```
Comment (domfournier) [inline geoapps/inversion/potential_fields/gravity/constants.py:125]: Can you double-check all the titles? Not sure we are consistant.

File: geoapps/inversion/potential_fields/gravity/constants.py [hunk 9]
```diff
@@ -195,6 +193,17 @@
 }
 
 forward_ui_json = {
+    "starting_model": {
+        "association": ["Cell", "Vertex"],
```
Comment (domfournier) [inline geoapps/inversion/potential_fields/gravity/constants.py:197]:
```suggestion
        "association": "Cell",
```

File: geoapps/inversion/potential_fields/magnetic_scalar/constants.py [hunk 10]
```diff
@@ -201,6 +199,17 @@
 }
 
 forward_ui_json = {
+    "starting_model": {
+        "association": ["Cell", "Vertex"],
```
Comment (domfournier) [inline geoapps/inversion/potential_fields/magnetic_scalar/constants.py:203]:
```suggestion
        "association": "Cell",
```

File: geoapps/inversion/potential_fields/magnetic_vector/constants.py [hunk 11]
```diff
@@ -208,6 +206,17 @@
 }
 
 forward_ui_json = {
+    "starting_model": {
+        "association": ["Cell", "Vertex"],
```
Comment (domfournier) [inline geoapps/inversion/potential_fields/magnetic_vector/constants.py:210]:
```suggestion
        "association": "Cell",
```

**Review by domfournier — CHANGES_REQUESTED** [review]
Would be so nice to have to the base inversion constants to handle the core ui part... anyway. Looking good overall. Few edits and we should be good.

File: geoapps/inversion/electricals/direct_current/two_dimensions/constants.py (hunk 1 above)
Comment (benk-mira) [reply geoapps/inversion/electricals/direct_current/two_dimensions/constants.py]: yep, I took a shortcut.  Removed now.

File: geoapps/inversion/electricals/induced_polarization/three_dimensions/constants.py (hunk 2 above)
Comment (benk-mira) [reply geoapps/inversion/electricals/induced_polarization/three_dimensions/constants.py:17]: Done

File: geoapps/inversion/electricals/induced_polarization/three_dimensions/constants.py [hunk 12]
```diff
@@ -78,30 +80,31 @@
...
     "run_command": "geoapps.inversion.driver",
-    "run_command_boolean": False,
     "conda_environment": "geoapps",
     "distributed_workers": None,
     "chargeability_channel_bool": True,
 }
 
 forward_defaults = {
-    "title": "SimPEG Induced Polarization Forward",
+    "title": "SimPEG Induced Polarization 3d Forward",
```
[the remark this answers is not in the document]
Comment (benk-mira) [reply geoapps/inversion/electricals/induced_polarization/three_dimensions/constants.py:95]: Done

File: geoapps/inversion/electricals/induced_polarization/two_dimensions/constants.py (hunk 7 above)
Comment (benk-mira) [reply geoapps/inversion/electricals/induced_polarization/two_dimensions/constants.py]: Done

**Review by domfournier — APPROVED** [review]
good to go

**Comment by domfournier** [comment]
@benk-mira Merge?

**Pull request merged**